scripts/ci/build_emscripten.sh

```diff
@@ -61,12 +62,20 @@ then
 	echo -n "$CIRCLE_SHA1" >commit_hash.txt
 fi
 
+# Disable warnings for unqualified "move()" calls, introduced and enabled by
+# default in clang-16 which is what the emscripten docker image uses.
+# Additionally, disable the warning for unknown warnings here, as this script is
+# also used with earlier clang versions.
+CMAKE_CXX_FLAGS="-Wno-unqualified-std-cast-call -Wno-unknown-warning-option"
```

Comment:
Can you add the TODO comment I mentioned earlier? I really think we need something that clearly indicates when the hack can be removed.

Reply:
The hack can be removed literally right after this PR and I will do that once it's merged <3 so I don't think we need the comment